common.sh

```diff
@@ -126,3 +126,6 @@ if [ ! -d "$WORKING_DIR" ]; then
 fi
 sudo chown "${USER}:${USER}" "$WORKING_DIR"
 chmod 755 "$WORKING_DIR"
+
+# Allow overriding openshift-install binary location, to e.g. build from git
+export OPENSHIFT_INSTALLER=${OPENSHIFT_INSTALLER:-"ocp/openshift-install"}
```

Comment:
want to document this in `config_example.sh` ?

Reply:
Sure, will do